addons/project/views/project_update_views.xml

```diff
@@ -53,10 +53,7 @@
                         </page>
                     </notebook>
                 </sheet>
-                <div class="oe_chatter d-print-none">
-                    <field name="message_follower_ids" options="{'post_refresh':True}" groups="base.group_user"/>
-                    <field name="message_ids"/>
-                </div>
+                <chatter class="d-print-none" reload_on_follower="1"/>
```

Comment:
to be verified but I don't think we forward class attribute?

Reply:
It's forwarded on chatter in this view.